internal/cfgplugins/interface.go

```diff
@@ -87,50 +89,86 @@ func ConfigOpticalChannel(t *testing.T, dut *ondatra.DUTDevice, och string, freq
 // ConfigOTNChannel configures the OTN channel.
 func ConfigOTNChannel(t *testing.T, dut *ondatra.DUTDevice, och string, otnIndex, ethIndex uint32) {
 	t.Helper()
-	gnmi.Replace(t, dut, gnmi.OC().TerminalDevice().Channel(otnIndex).Config(), &oc.TerminalDevice_Channel{
-		Description:        ygot.String("OTN Logical Channel"),
-		Index:              ygot.Uint32(otnIndex),
-		LogicalChannelType: oc.TransportTypes_LOGICAL_ELEMENT_PROTOCOL_TYPE_PROT_OTN,
-		TribProtocol:       oc.TransportTypes_TRIBUTARY_PROTOCOL_TYPE_PROT_400GE,
-		Assignment: map[uint32]*oc.TerminalDevice_Channel_Assignment{
-			0: {
-				Index:          ygot.Uint32(0),
-				OpticalChannel: ygot.String(och),
-				Description:    ygot.String("OTN to Optical Channel"),
-				Allocation:     ygot.Float64(400),
-				AssignmentType: oc.Assignment_AssignmentType_OPTICAL_CHANNEL,
+	if deviations.OTNChannelTribUnsupported(dut) {
+		gnmi.Replace(t, dut, gnmi.OC().TerminalDevice().Channel(otnIndex).Config(), &oc.TerminalDevice_Channel{
+			Description:        ygot.String("OTN Logical Channel"),
+			Index:              ygot.Uint32(otnIndex),
+			LogicalChannelType: oc.TransportTypes_LOGICAL_ELEMENT_PROTOCOL_TYPE_PROT_OTN,
+			Assignment: map[uint32]*oc.TerminalDevice_Channel_Assignment{
+				0: {
```

Comment:
does this need to be "1" to match the index?

Reply:
Not necessary. It's the inner index which needs to start from 1 instead of 0.